hs_core/models.py

```diff
@@ -961,6 +961,36 @@ def __unicode__(self):
         return self.value
 
 
+class FundingAgency(AbstractMetaDataElement):
+    term = 'FundingAgency'
+    agency_name = models.TextField(null=False)
+    award_title = models.TextField(null=True, blank=True)
+    award_number = models.TextField(null=True, blank=True)
+    agency_url = models.URLField(null=True, blank=True)
+
+    class Meta:
+        unique_together = ("content_type", "object_id", "award_number", "agency_name")
```

Comment:
@hyi - if multiple awards are listed, each award should only be listed once. It is theoretically possible (although unlikely) for the same award number to be used by two different agencies for two different awards. So, the rule should be:  where multiple awards are listed, the combination of award number/agency name for each award should be unique.